common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -571,6 +571,21 @@ object Utils extends Logging {
     // scalastyle:on classforname
   }
 
+  def instantiateMasterEndpointResolver[T](
+      className: String,
+      conf: CelebornConf,
+      isWorker: Boolean): T = {
+    try {
+      val cls = classForName(className)
+      cls.getConstructor(classOf[CelebornConf], java.lang.Boolean.TYPE)
+        .newInstance(conf, java.lang.Boolean.valueOf(isWorker))
+        .asInstanceOf[T]
```

Comment:
```suggestion
      DynConstructors.builder().impl(className, classOf[CelebornConf], java.lang.Boolean.TYPE)
        .build[T]()
        .newInstance(conf, java.lang.Boolean.valueOf(isWorker))
```

Reply:
LGTM, suggestion is missing the import so not committing from UI. Will fix this in the next commit.